go/git/get.go

```
@@ -183,17 +183,20 @@ func getMetadataInner(ctx context.Context, g *libkb.GlobalContext, folder *keyba
 			return nil, fmt.Errorf("unrecognized variant of GitLocalMetadataVersioned: %#v", version)
 		}
 
-		// Load UPAKs to get the device name.
-		upak, _, err := g.GetUPAKLoader().LoadV2(libkb.LoadUserArg{Name: responseRepo.LastModifyingUsername})
+		// Load UPAKs to get the last writer username and device name.
+		lastWriterUPAK, _, err := g.GetUPAKLoader().LoadV2(libkb.LoadUserArg{UID: responseRepo.LastModifyingUID})
+		if err != nil {
+			return nil, err
+		}
```

Woops, this was an ignored error value before.